chart/tests/test_scheduler.py

```diff
@@ -174,3 +174,27 @@ def test_logs_persistence_changes_volume(self, log_persistence_values, expected_
         assert {"name": "logs", **expected_volume} == jmespath.search(
             "spec.template.spec.volumes[1]", docs[0]
         )
+
+    def test_scheduler_resources_are_configurable(self):
```

Comment:
Can you add this for `webserver` and `workers` too please

Reply:
Done!